openedx/features/content_tagging/fixtures/system_defined.yaml

```diff
@@ -0,0 +1,33 @@
+- model: oel_tagging.taxonomy
+  pk: -1
```

Comment:
Shouldn't this fixture (for taxonomy -1, languages) live in openedx-learning? Because the ID `-1` is already hard-coded in there for the tags, in `openedx_tagging/core/tagging/fixtures/language_taxonomy.yaml`